dist/common/scripts/scylla_cpuset_setup

```diff
@@ -26,12 +26,18 @@ if __name__ == '__main__':
         parser.print_help()
         sys.exit(1)
 
-    cfg = sysconfig_parser('/etc/scylla.d/cpuset.conf')
-    cfg.set('CPUSET', '{cpuset}{smp}'.format( \
-            cpuset='--cpuset {} '.format(args.cpuset) if args.cpuset else '', \
-            smp='--smp {} '.format(args.smp) if args.smp else '' \
-            ))
-    cfg.commit()
-
-    if os.path.exists('/etc/scylla.d/perftune.yaml'):
-        os.remove('/etc/scylla.d/perftune.yaml')
+    cpuset = smp = None
+    try:
+        cfg = sysconfig_parser('/etc/scylla.d/cpuset.conf')
+        line = cfg.get('CPUSET')
+        cpuset_args = parser.parse_args(line.split())
+        cpuset = cpuset_args.cpuset
+        smp = cpuset_args.smp
+    except:
+        pass
+    if cpuset != args.cpuset or smp != args.smp:
```

Comment:
this I suppose is sensitive to what user provides with what we write
is that input format validated somehow? (or trimmed?)

Reply:
@tarzanek I added cpuset.conf parsing code by re-using argparse, since the format is exactly same as program argument.
Since we applied argparse, it will trim whitespaces, but it won't validate values itself since we don't have cpuset format validator in the script (the format is described at https://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS, it's not just list of cpu numbers, bit more complicated).

For --smp option, probably we should add type=int for parser.add_argument, since it simple numeric.

Reply:
We actually have regex pattern in seastar: https://github.com/scylladb/seastar/blob/master/src/core/resource.cc#L48